code/modules/clothing/clothing.dm

```diff
@@ -139,16 +139,27 @@
 	if(damaged_clothes)
 		user <<  "<span class='warning'>It looks damaged!</span>"
 
+	if(user.robustness_scanner) //Mob has a robustness scanner active.
+		var/msg = "*--------* <BR>"
 
-	list_clothing_armour(user, armor)
+		if(armor)
+			msg += "<span class='notice'>Armor:</span><BR>"
+			msg += "[list_clothing_armour(user, armor)]<BR>"

```

Comment:
this shows up in the wrong place and i'm not sure how to fix it
![](https://puu.sh/tTMZ6/13e18cf627.png)